### PR TITLE
adds an environment variable for INPUT_BATCH_SIZE

### PR DESCRIPTION
**Pull request by xiaoling**
cf HazyResearch/ddlog@384488ceb6a5c3ac8bd5a9a96c8c5861be366ffa

### CODE REVIEW

File: shell/deepdive-run [hunk 1]
```diff
@@ -108,6 +108,9 @@ fullConfig=$run_dir/deepdive.conf
         echo 1
     ) 2>/dev/null)}
     export PARALLELISM
+    
+    # set INPUT_BATCH_SIZE env var 
+    export INPUT_BATCH_SIZE=10000
```
Comment (netj) [inline shell/deepdive-run]: Always settings this prevents the user from overriding the value.  Please fix it to respect user's environment.

File: shell/deepdive-run [hunk 2]
```diff
@@ -108,6 +108,9 @@ fullConfig=$run_dir/deepdive.conf
         echo 1
     ) 2>/dev/null)}
     export PARALLELISM
+    
+    # set INPUT_BATCH_SIZE env var 
+    ! [[ -e "$INPUT_BATCH_SIZE" ]] || export INPUT_BATCH_SIZE=10000
```
Comment (netj) [inline shell/deepdive-run]: First, when `INPUT_BATCH_SIZE` is not defined, the entire script aborts.  Second, this test with `-e` is checking whether there *exists* a file named with that value which doesn't make sense.

Here's the right way to do this:
```bash
: ${INPUT_BATCH_SIZE:=10000}
export INPUT_BATCH_SIZE
```

Comment (xiaoling) [inline shell/deepdive-run]: You're right. Sorry my bash is a bit rusty. I should've been more careful.

Comment (netj) [inline shell/deepdive-run]: No worries.  Can you also update the ddlog submodule to point to the commit relevant to this?

**Comment by netj** [comment]
LGTM

**Pull request merged**